src/main/java/org/cactoos/scalar/AvgOf.java

```diff
@@ -83,16 +83,13 @@ public AvgOf(final Integer... src) {
             }
```

Comment:
@allanws First of all, I think it a good idea look at the @svendiedrichsen [suggestion](https://github.com/svendiedrichsen/cactoos/commit/358b7f19469669df00eeea4c2ec3c5fbffd5ac8e) and [comments](https://github.com/yegor256/cactoos/issues/532#issuecomment-356689894) about this issue.